src/engine/sync/internalclock.cpp

```diff
@@ -87,7 +90,8 @@ double InternalClock::getBeatDistance() const {
 }
 
 void InternalClock::setMasterBeatDistance(double beatDistance) {
-    //qDebug() << "InternalClock::setBeatDistance" << beatDistance;
+    //qDebug() << "InternalClock::setMasterBeatDistance" << beatDistance;
+    m_bClockUpdated.store(true);
```

Comment:
This causes a warning, can you use `atomicStoreRelaxed` (or `atomicStoreAcquire`, whatever you need) from `util/compatibility.h` instead please?
https://github.com/mixxxdj/mixxx/blob/80a2a0bad0e8fa661cfc1bf1dc077b64b0df1d74/src/util/compatibility.h#L183

Reply:
fixed

Reply:
hmmmm, I feel like fetchandstoreordered would be more appropriate, what do you think?